app/assets/stylesheets/dashboard_sidebar.scss

```diff
@@ -0,0 +1,25 @@
+.sidebar{
+    background-color:#fff;
+    background-repeat: repeat;
+    padding:0;
+    min-height:100% !important;
+    position:relative;
+    border-right: 1px solid #eee;
+    border-top: 1px solid #eee;
+}
+
+html, body {height: 100%}
```

Comment:
lets go ahead and expand single liners to look like the rest. whitespace is free and we compress it anyway.